app/build.gradle

```diff
@@ -47,6 +47,12 @@ android {
             applicationIdSuffix ".debug"
             versionNameSuffix ".debug"
         }
+        benchmark {
+            signingConfig signingConfigs.debug
+            matchingFallbacks = ['release']
+            manifestPlaceholders["sentryDsn"] = ""
```

Comment:
Why is this needed here?

Reply:
All this stuff is IDE generated i just added some bits to make solve some errors